src/ops/arithmetic.rs

```diff
@@ -0,0 +1,875 @@
+use crate::array::Array;
+use crate::stream::StreamOrDevice;
+use mlx_macros::default_device;
+
+impl Array {
+    /// Element-wise absolute value.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// use mlx::Array;
+    /// let array = Array::from_slice(&[1i32, 2, -3, -4, -5], &[5]);
+    /// let mut result = array.abs();
+    ///
+    /// result.eval();
+    /// let data: &[i32] = result.as_slice();
+    /// // data == [1, 2, 3, 4, 5]
```

Comment:
Maybe use `assert_eq!()` in the doc example?

Reply:
We decided not to, after you looked into it